xml/System.ServiceModel.Channels/MsmqBindingElementBase.xml

```diff
@@ -457,7 +457,11 @@
   
  ]]></format>
         </remarks>
-        <exception cref="T:System.ArgumentOutOfRangeException">The value is not infinite and larger than Int32.MaxValue milliseconds.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">The value is less than zero.
+        
+-or-
+
+The value is not infinite and larger than Int32.MaxValue milliseconds.</exception>
```

Comment:
nit for a future fix: "The value is not infinite and is larger than Int32.MaxValue milliseconds."
There are a few occurrences of this.